### PR TITLE
Fix Links

### PR DESCRIPTION
**Pull request by Zehir**
Replace absolute img link to relative
Fix one doc link

### CODE REVIEW

**Comment by Zehir** [comment]
@Benjamin-Dobell this update works on my repo but maybe you want to check before merging ?

File: docs/index.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 Welcome to **Lua Scripting** in Tabletop Simulator! Scripting is an advanced feature and it’s expected you already know something about it. However, for complete novices, check out this [beginners scripting guide to Lua](http://steamcommunity.com/sharedfiles/filedetails/?id=714904631) created by MrStump, to help you along the way.
 
-![Demonstration Of Game](/img/intro.gif)
+![Demonstration Of Game](img/intro.gif)
```
Comment (Benjamin-Dobell) [inline docs/index.md:5]: Should this one be `../` as well?

Comment (Benjamin-Dobell) [reply docs/index.md:5]: Nope. It's correct. Just pulled down the branch and verified 👍

Comment (Zehir) [reply docs/index.md:5]: Yes @Benjamin-Dobell it's the index page so at the same level of img directory.

**Pull request merged**